client/src/components/Card.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import styled from "styled-components/macro";
+import restaurant from "../stories/assets/restaurant.png";
```

Comment:
i would move the assets folder to src

Reply:
Good idea, added to my to do list now! 🦾 i'll do it when i refactore my code, thank you!